statistics_collector_lib/include/collector/collector.hpp

```diff
@@ -24,7 +24,7 @@
 
 #include "rcpputils/thread_safety_annotations.hpp"
 
-namespace system_metrics_collector
+namespace collector
```

Comment:
better to match the director structure collector -> libsystem_metrics_collector

Reply:
The directory structure has changed enough for this comment to no longer be applicable.